GLOSSARY.md

```diff
@@ -0,0 +1,40 @@
+# GitOps Glossary
+
+This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other supporting documents in this repository.
+
+- ## Continuous
+
+    "Continuous" is intended to match the industry standard term: [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
+
+- ## Declarative Description
+
+    A configuration that describes the desired operating state of a system without specifying procedures for how that state will be acheived. This separates configuration (the desired state) from the implementation (commands, API calls, scripts etc.) used to achieve that state.
+
+- ## Desired State
+
+    The aggregate of all configiuration data needed to start, bootstrap, recreate, run, and recover a system. This does not include data stored within a system, eg. database contents.
+
+- ## Drift
+
+    When a system's actual state changes for any reason other than its [desired state](#desired-state)declarations having changed. When this occurs, the system has drifted from its desired state.
+
+- ## Reconciliation
+
+    The process of ensuring the actual state of a system matches its [desired state](#desired-state) declarations.
+    Contrary to "CIops", any divergence between the two will trigger reconciliation, regardless of where changes occured.
```

Comment:
```suggestion
    Contrary to "CI Ops", any divergence between the two will trigger reconciliation, regardless of where changes occurred.
```
1. Related, https://github.com/open-gitops/documents/pull/8
• "CIOps" -> "CI Ops"
2. Fix typo:
• occured -> occurred

Reply:
Hi @lloydchang thanks!

Everyone:
- We could use some more group feedback about @lloydchang 's point 1 above. Here is the existing discussion on that for (slightly) more context: https://github.com/open-gitops/documents/pull/8

@lloydchang:
- Point 2, yes! thanks for the typo fix. Can you make a separate code suggestion just for the typo, since that does not need further discussion?

Reply:
```suggestion
    Contrary to "CIops", any divergence between the two will trigger reconciliation, regardless of where changes occurred.
```
2. Fix typo _(only)_:
• occured -> occurred
_(as requested by @scottrigby)_

Reply:
A. @todaywasawesome suggested "traditional CI/CD" immediately above.
B. @scottrigby hinted at "Operations driven by Continuous Integration" in https://github.com/open-gitops/documents/pull/8#pullrequestreview-687559861
Either suggestion or hint works for me because my primary objective of replacing "`CIOps`" is to prevent anyone else from accidentally searching for "`Clops`" like I did.

Reply:
Ok, so we changed the language to CI/CD and made it a bit more clear. I don't know that it's 100% for everyone but lets resolve it for now as it's not a blocker for v1.